diff --git a/Cargo.toml b/Cargo.toml
index fda81c9..5a151dd 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -40,6 +40,7 @@ ctrlc = { version = "3.2.3", features = ["termination"], optional = true }
 derive_builder = "0.13.0"
 shadow-rs = { version = "0.26.1", optional = true }
 sysinfo = { version = "0.30.5", optional = true }
+time = "0.3.35"
 thiserror = "1.0.56"
 
 [target.'cfg(windows)'.dependencies.windows]
